src/components/Picker.js

```diff
@@ -2,7 +2,7 @@ import _ from 'underscore';
 import React, {PureComponent} from 'react';
 import {View} from 'react-native';
 import PropTypes from 'prop-types';
-import Picker from './Picker';
+import NativePicker from './NativePicker';
```

Comment:
Ok. Let's do one thing. 
2. Rename it to BasePicker and move the whole NativePicker folder to Picker folder.
3. Move Picker.js to Picker/index.js
4. Create Picker Folder.